libs/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -85,6 +114,10 @@ export class TriggerEvent {
         organizationId: mappedCommand.organizationId,
       });
 
+      const discoverWorkflow = await this.queryDiscoverWorkflow(command);
```

Comment:
This should throw an error if not successfully, and also suggest to move this to the /events/trigger API call level so we can throw an HTTP error

Reply:
done